addons/report/models/report.py

```diff
@@ -194,6 +194,8 @@ def get_pdf(self, docids, report_name, html=None, data=None):
         # Minimal page renderer
         view_obj = self.env['ir.ui.view']
         render_minimal = partial(view_obj.with_context(context).render_template, 'report.minimal_layout')
+        # Page header renderer - to show the link in alert box if address and account number is not configured. This should be added in header but because of some issue on git hub, adding here.
+        render_minimal_page_header = partial(view_obj.with_context(context).render_template, 'report.external_header_page')
```

Comment:
Why is it done here and not directly managed into the report templates ? Makes no sense to have several reports concatenated.